lib/plain/endpoints/common-types.ts

```diff
@@ -3,6 +3,7 @@ import { QueryOptions, PaginationQueryOptions } from '../../common-types'
 export type GetSpaceParams = { spaceId: string }
 export type GetSpaceEnvironmentParams = { spaceId: string; environmentId: string }
 export type GetOrganizationParams = { organizationId: string }
+export type GetTeamParams = { organizationId: string; teamId: string }
```

Comment:
not exactly related to this PR, but why do we name these types as Get*Params? I would suggest just SpaceParams for ex
Update: So I think you name it like that because they are params for the GET calls yes? 🤔

Reply:
@vbeleuta correct, types are present to ease re-use and naming is because the sole purpose is for typing the many get calls.